packages/mdc-toolbar/_mixins.scss

```diff
@@ -35,7 +35,8 @@
 }
 
 @mixin mdc-toolbar-icon-ink-color($color) {
-  @include mdc-theme-prop(color, $color);
+  @include mdc-theme-prop(color, $color); // for font icons
+  @include mdc-theme-prop(fill, $color); // for SVG icons
```

Comment:
I think it should be possible to do this centrally rather than upon each application of this mixin, by setting `fill: currentColor` in the `mdc-toolbar-icon_` mixin instead?

Also CC @williamernest in case he knows of any reason setting `fill: currentColor` might be problematic in any future planned toolbar instances